docs/start/integration.md

```diff
@@ -219,15 +225,15 @@ You can fetch the canonical block for a given layer using [`MeshService.LayersQu
 
 #### Multiple Blocks
 
-Unless security assumptions are violated, there will be zero or one block per layer. Examples of security assumption violations are a large number (> 1/3) of miner weight colluding and equivocating, or a long-lived network partition (longer than an epoch) followed by a rejoin.
+Unless security assumptions are violated, there will be zero or one block per layer. Examples of security assumption violations are a large number (> 1/3) of smesher weight colluding and equivocating, or a long-lived network partition (longer than an epoch) followed by a rejoin.
 
 According to the Spacemesh protocol, only one block per layer may be considered canonical and only the transactions in that block may be included in the mesh. For more information on how this works in the protocol, see [explorer-backend#92](https://github.com/spacemeshos/explorer-backend/issues/92#issuecomment-1841293498).
 
-Unfortunately block validity is not currently exposed in the API (we're [working on it](https://github.com/spacemeshos/api/issues/287)). It can be manually read from the node database.
+Unfortunately, block validity is not currently exposed in the API (we're [working on it](https://github.com/spacemeshos/api/issues/287)). However, it can be manually read from the node database.
 
 ### Fetching Account Balance
 
-Use [`GlobalStateService.Account`](https://pkg.go.dev/github.com/spacemeshos/go-spacemesh@v1.2.1/api/grpcserver#GlobalStateService.Account). Balance is denominated in smidge, where 1 SMH = 1 billion smidge (1e9 smidge). `stateCurrent` shows the current account balance; `stateProjected` includes transactions that have been mined into blocks but not yet applied to state (because the layer hasn't been finalized).
+To get the balance of an account, use [`GlobalStateService.Account`](https://pkg.go.dev/github.com/spacemeshos/go-spacemesh@v1.2.1/api/grpcserver#GlobalStateService.Account). Balance is denominated in a **smidge**, where 1 SMH = 1 billion smidge (1e9 smidge). `stateCurrent` shows the current account balance while `stateProjected` includes transactions that have been mined into blocks but not yet applied to state (because the layer has not yet been finalized).
```

Comment:
```suggestion
To get the balance of an account, use [`GlobalStateService.Account`](https://pkg.go.dev/github.com/spacemeshos/go-spacemesh@v1.2.1/api/grpcserver#GlobalStateService.Account). Balance is denominated in **smidge**, where 1 SMH = 1 billion smidge (1e9 smidge). `stateCurrent` shows the current account balance while `stateProjected` includes transactions that have been mined into blocks but not yet applied to state (because the layer has not yet been finalized).
```